models/account/account.go

```diff
@@ -0,0 +1,106 @@
+package account
+
+import (
+	"database/sql"
+	"fmt"
+	"strconv"
+
+	"github.com/b0d0nne11/scroll-test-project/db"
+	"github.com/mailgun/scroll"
+)
+
+type Account struct {
+	ID   int64
+	Name string
+}
+
+func New(name string) *Account {
+	return &Account{
+		Name: name,
+	}
+}
+
+func (a *Account) Save() error {
```

Comment:
Having a `save` method on the model might result in some bad circumstances in concurrent programs. Please ping me I'll explain it in details.